Jenkinsfile

```diff
@@ -4,66 +4,34 @@
 def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 def target_backends = ['aws', 'gce', 'docker', 'k8s-local-kind-aws']
-
-def runSctProvisionTest(String backend){
-    if (currentBuild.result != null) {
-        return
-    }
-    echo "Going to Provision test on ${backend} for Pull Request ID: ${env.CHANGE_ID}"
-    checkout scm
-    def distro_cmd = ""
-    def test_cmd = "run-test"
-    def test_name = "longevity_test.LongevityTest.test_custom_time"
-
-    def sct_config_files = "test-cases/PR-provision-test.yaml"
+def sct_runner_backends = ['aws', 'gce', 'k8s-local-kind-aws']
+
+def createRunConfiguration(String backend) {
+
+    def configuration = [
+        backend: backend,
+        test_name: 'longevity_test.LongevityTest.test_custom_time',
+        test_config: 'test-cases/PR-provision-test.yaml',
+        availability_zone: 'a',
+        scylla_version: '4.6.3',
```

Comment:
Maybe 5.0.0?

Reply:
first stabilize it on this release, then we could move to 5.0 on it's own PR